third-party/unittest/googletest/README.LLVM

```diff
@@ -1,22 +1,21 @@
 LLVM notes
 ----------
 
-This directory contains Google Test 1.10.0, with all elements removed except for
-the actual source code, to minimize the addition to the LLVM distribution.
+This directory contains Google Test 1.14.0,
+revision `f8d7d77c06936315286eb55f8de22cd23c188571`, with all elements removed
+except for the actual source code, to minimize the addition to the LLVM
+distribution.
 
 Cleaned up as follows:
 
 # Remove all the unnecessary files and directories
 $ rm -f CMakeLists.txt configure* Makefile* CHANGES CONTRIBUTORS README README.md .gitignore
 $ rm -rf build-aux cmake codegear m4 make msvc samples scripts test xcode docs
-$ rm -f `find . -name \*\.pump`
 $ rm -f src/gtest_main.cc
 
 # Put the license in the consistent place for LLVM.
 $ mv LICENSE LICENSE.TXT
 
 Modified as follows:
-* Added support for NetBSD, Minix and Haiku.
 * Added raw_os_ostream support to include/gtest/internal/custom/gtest-printers.h.
 * Added StringRef support to include/gtest/internal/custom/gtest-printers.h.
```

Comment:
Also include/gtest/gtest-message.h and include/gtest/gtest-printers.h. Probably worth saying something like "(see uses of llvm_gtest)" to help the next person doing a roll.

Reply:
I added a description for the these two headers.